traits.yml

```diff
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=../../python/openassetio_traitgen/schema.json
+# yamllint disable-line rule:document-start
+package: openassetio-mediacreation
+
+description:
+    Well-known Traits and Specifications for use in OpenAssetIO hosts and managers.
+
+traits:
+    timeline:
+        description: Traits related to timelines.
+        members:
+            Timeline:
+                description: This trait characterizes a collection of tracks that evaluate
```

Comment:
[consider] Use `>` then you can move the text to the next line, giving a bit more space before wrapping. 

 Note that `|` could be used too, but then you're double line-wrapping, here and in the render, so chances are it won't look right.

Note that you need a double line break if you want to have a single line gap in the rendered output when using `>`.

Reply:
Took your advice, use `>` in all paragraph cases and add double line seperators. [8c6c35e](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/15/commits/8c6c35e7f1ad1f660a4efa481b4a98096a7f68b4)